### PR TITLE
[Excel] (Custom Functions) Adding coauthoring note

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[29f0d9d](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/29f0d9df0fc1790c66112d1d7b8c1581f156bd74)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[29f0d9d](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/29f0d9df0fc1790c66112d1d7b8c1581f156bd74)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[29f0d9d](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/29f0d9df0fc1790c66112d1d7b8c1581f156bd74)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-coauth/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902082222070840-742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=J9IV1pi2Le%2FsAWNrks2t8KbvsomTugHamWzMYm1RUFs%3D&st=2019-02-08T22%3A18%3A20Z&se=2019-03-11T22%3A23%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -327,6 +327,11 @@ function refreshTemperature(thermometerID){
 }
 ```
 
+## Co-Authoring
+Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature is supported by custom functions. If your workbook uses a custom function, your sharing partner will be prompted to install the custom function's add-in in order to display the results. Once both people sharing the document have installed the add-in, all custom function behavior should work normally.
```
Comment (davidchesnut) [inline docs/excel/custom-functions-overview.md]:
```suggestion
Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
```

Comment (davidchesnut) [reply docs/excel/custom-functions-overview.md]: Just a few consistency recommendations like using "colleague" similar to support docs like here https://support.office.com/en-us/article/collaborate-on-excel-workbooks-at-the-same-time-with-co-authoring-7152aa8b-b791-414c-a3bb-3024e46fb104

File: docs/excel/custom-functions-overview.md [hunk 2]
```diff
@@ -327,6 +327,11 @@ function refreshTemperature(thermometerID){
...
+
+For more information on coauthoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```
Comment (davidchesnut) [inline docs/excel/custom-functions-overview.md]:
```suggestion
For more information on co-authoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d66a933](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/d66a933796e7e4a77daa3e2361ae27538957e7c9)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d66a933](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/d66a933796e7e4a77daa3e2361ae27538957e7c9)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

File: docs/excel/custom-functions-overview.md [hunk 3]
```diff
@@ -327,6 +327,11 @@ function refreshTemperature(thermometerID){
 }
 ```
 
+## Co-Authoring
+Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
+
+For more information on co-authoring, see [About CoAuthoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```
Comment (mscharlock) [inline docs/excel/custom-functions-overview.md]:
```suggestion
For more information on co-authoring, see [About Co-Authoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d66a933](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/d66a933796e7e4a77daa3e2361ae27538957e7c9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-coauth/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902131915177933-742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lnHU3bFOdpQoMad%2BeleIxgAYwkCjSLOc3Op17xt3bco%3D&st=2019-02-13T19%3A11%3A32Z&se=2019-03-16T19%3A16%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[78018a2](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/78018a2b7cd881ee79c5e7b04018aa63016a8a65)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[78018a2](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/78018a2b7cd881ee79c5e7b04018aa63016a8a65)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[78018a2](https://github.com/OfficeDev/office-js-docs-pr/pull/742/commits/78018a2b7cd881ee79c5e7b04018aa63016a8a65)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-coauth/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902131916287600-742%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JrR2enAOVrtV6EBHa2AwFWo57s9gzVETei%2F6aUdY2tE%3D&st=2019-02-13T19%3A12%3A51Z&se=2019-03-16T19%3A17%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**